css/style.css

```diff
@@ -0,0 +1,447 @@
+* {
+  box-sizing: border-box;
+  margin: 0;
+  padding: 0;
+}
+
+.navbar {
+  width: 100%;
+  height: auto;
+  padding: 10px 10px 20px 10px;
+  display: flex;
+  flex-direction: column;
+  justify-content: space-between;
+  position: fixed;
+  top: 0;
+  left: 0;
+}
+
+.menu-icon {
+  padding: 30px 10px 10px 20px;
+  width: 43px;
+  height: auto;
+}
+
+.m-menu {
+  width: 30px;
+  margin-bottom: 3px;
+  margin-bottom: 10%;
+  border: 3px solid #272a31;
+}
+
+.head-line {
+  display: none;
+  width: 100%;
+  height: 50px;
+  position: fixed;
+  top: 0;
+  left: 0;
+  background-color: #3e3e3e;
+  flex-direction: row;
+  justify-content: flex-end;
+  padding: 20px 20px;
+}
+
+.head-img {
+  width: 25px;
+  height: auto;
+  display: flex;
+  align-self: center;
+  margin-right: 20px;
+}
+
+.head-link {
+  color: #fff;
+  margin-right: 20px;
+}
+
+.menu-ul {
+  list-style: none;
+  display: none;
+  flex-direction: column;
+  width: 100%;
+  height: auto;
+  padding-left: 2%;
+  position: fixed;
+  top: 0;
+  left: 0;
+  background-color: #fff;
+  margin-bottom: 20%;
+}
+
+.close {
+  width: 50px;
+  height: 50px;
+  margin-top: 20px;
+}
+
+.header-logo {
+  display: none;
+}
+
+.links {
+  padding-top: 5px;
+  margin: 0 0 10px 10px;
+}
+
+.menu-links {
+  text-decoration: none;
+  color: #272a31;
+}
+
+.menu-ul.active {
+  display: flex;
+}
+
+.pg1-sec {
+  background-image: url("../Images/background.png");
+  height: auto;
+  padding: 80px 50px;
+  width: 100%;
+  padding-top: 210px;
+}
+
+@font-face {
+  font-family: cocogoose;
+  src: url(../Images/fonts/Cocogoose.ttf);
+}
+
+.hello-world {
+  font-size: 25px;
+  font-family: 'Lato', sans-serif;
+  color: #ec5242;
+  margin-bottom: 10px;
+}
+
+.event-name {
+  font-size: 30px;
+  font-family: 'cocogoose', sans-serif;
+  width: 75%;
+  color: #ec5242;
+}
+
+.event-dis {
+  font-family: 'Inter', sans-serif;
+  font-size: 16px;
+  border: 6px solid #fff;
+  padding: 20px 20px;
+  margin-top: 20px;
+}
+
+.event-start {
+  display: flex;
+  justify-content: center;
+  font-size: 24px;
+  padding: 20px 20px;
+  margin-top: 20px;
+}
+
+.user-sec {
+  background-image: url("../Images/user-sec.jpg");
+  width: 100%;
+  height: auto;
+  padding: 20px 20px;
+}
```

Comment:
- [ OPTIONAL] For this section the background is looking okay but you can improve it to look like the design check images below 👇🏻  🤔  I think you can change your background image instead use an image with a `background-repeat: repeat;` property. What do you think?

- [Cindy Shin](https://www.behance.net/adagio07)[Cindy Shin](https://www.behance.net/adagio07) design:

![og](https://user-images.githubusercontent.com/44745855/132837558-b3e87012-6b81-43f7-9b74-dd6a88f8adab.png)

- Your design:

![25main](https://user-images.githubusercontent.com/79058364/175758889-c6ec3158-0251-4142-91ab-b3f27daab0d2.png)

- [OPTIONAL] Kindly add the `SEE THE WHOLE PROGRAM` text provide for this project section button in the desktop view 👍 

- Required design:
![Annotation 2022-05-24 194142](https://user-images.githubusercontent.com/85113567/170111416-153bb47c-ec03-44a0-a0f4-f045194f4952.jpg)